packages/cli/tests/images/build.js

```diff
@@ -73,10 +75,8 @@ exports.prerender.heads.route66 = `
 	<link rel="preload" href="\\/route-route66\\.chunk\\.\\w{5}\\.js" as="script">
 	<link rel="preload" href="\\/route-route66\\.chunk\\.\\w{5}\\.css" as="style">
 	<link rel="shortcut icon" href="\\/favicon\\.ico">
-	<link href=\\"/bundle.\\w{5}.css\\" rel=\\"preload\\" as=\\"style\\" onload=\\"this.rel='stylesheet'\\"><noscript>
-		<link rel=\\"stylesheet\\" href=\\"\\/bundle.\\w{5}.css\\"><\\/noscript>
-	<style>html{padding:0;}<\\/style>
-<\\/head>
+	<link href=\\"/bundle.\\w{5}.css\\" rel="stylesheet">
```

Comment:
Why is the noscript tag removed here?
Critters should be adding it here as well right?

Reply:
Maybe the noscdipt tag is removed in critters2

Reply:
The tests started failing on next branch after we updated critters version

Reply:
maybe critters deemed it small enough? not sure

Reply:
@developit.. please comment

Reply:
also this does not have `rel=\\"preload\\" as=\\"style\\" onload=\\"this.rel='stylesheet'\\"` which makes it a blocking request

Reply:
! this is a valid bug.. the original content should be as is

Reply:
Critters does inline stylesheets below a threshold, but we don't have that enabled in configuration (https://github.com/GoogleChromeLabs/critters/blob/master/src/index.js#L75-L76). Seems like it's erroring out?